client/my-sites/plugins-wpcom/jetpack-plugins-panel.jsx

```diff
@@ -0,0 +1,220 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import {
+	identity,
+	find
+} from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import PluginIcon from 'my-sites/plugins/plugin-icon/plugin-icon';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import JetpackPluginItem from './jetpack-plugin-item';
+import SectionHeader from 'components/section-header';
+import SectionNav from 'components/section-nav';
+import NavTabs from 'components/section-nav/tabs';
+import NavItem from 'components/section-nav/item';
+import Search from 'components/search';
+import jetpackPlugins from './jetpack-plugins';
+
+class JetpackPluginsPanel extends Component {
+
+	static propTypes = {
+		siteSlug: PropTypes.string,
+		hasBusiness: PropTypes.bool,
+		hasPremium: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			selectedGroup: 'all',
+			searchTerm: '',
+		};
+	}
```

Comment:
we could get rid of this constructor by assigning to a property directly (it doesn't depend on `props`)

```js
class JetpackPluginsPanel extends Component {
	state = {
		selectedGroup: 'all',
		searchTerm: '',
	};
}
```